.github/scripts/workflow_rerun/rerunner.py

```diff
@@ -13,19 +14,20 @@
     args = get_arguments()
     run_id = args.run_id
     repository_name = args.repository_name
+    errors_file = args.errors_to_look_for_file
```

Comment:
Fixes argument name, was `error_to_look_for_file`, the script is not working in the master branch atm because of this bug.

Reply:
Added a test for it.